ext/openssl/tests/tlsv1.3_wrapper.phpt

```diff
@@ -0,0 +1,49 @@
+--TEST--
+tlsv1.3 stream wrapper
+--SKIPIF--
+<?php
+if (!extension_loaded("openssl")) die("skip openssl not loaded");
+if (!function_exists("proc_open")) die("skip no proc_open");
+?>
```

Comment:
it should also skip if compiled with older OpenSSL version than 1.1.1

Reply:
add check for openssl 1.1.1 skip if older

Reply:
added fix 4a4530b including minor versions, correctly skips if <1.1.1 is used now. 
Tested for 1.1.0 and 1.1.1